### PR TITLE
Express

### PR DESCRIPTION
**Pull request by hewitt-BBD**
basic express server for our website.

### CODE REVIEW

**Review by mmasehume-bbd — APPROVED** [review]
Shouldn't the html folder be renamed to game or something and that way we can add a js folder inside for the scripts?

File: html/pages/game_page.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <aside id="player-list rounded-container">
+        <section class="rounded-container placeholder"></section>
+    </aside>
+</section>
+<script>
+    var canvas = document.getElementById("canv");
+    var ctx = canvas.getContext("2d");
+    ctx.fillStyle = "#ffffff";
+    ctx.fillRect(0, 0, canvas.clientWidth, canvas.clientHeight);
+</script>
```
Comment (Marco-Gomes-BBD) [inline html/pages/game_page.html]: Add a new line at end of file
Goes for the other files as well.

In VSCode:
File > Preferences > Settings
Workspace
Text Editor.Files.insertFinalNewline = true
Text Editor.Files.trimFinalNewlines = true

Please commit the .vscode/settings.json.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (Marco-Gomes-BBD) [inline index.js:1]: Empty line at start of file?

File: index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+app.get('/game', function(req, res) {
+    res.sendFile(path.join(__dirname, 'html/pages/game_page.html'));
+});
+
+app.get('/player-select', function(req, res) {
+    res.sendFile(path.join(__dirname, "html/pages/player_select.html"))
+});
+
+app.listen(port);
+console.log('Server started at http://localhost:' + port);
```
Comment (Marco-Gomes-BBD) [inline index.js]: Rather use a callback for this, then we know the server is up.
```suggestion
app.listen(port, () => {
    console.log('Server started at http://localhost:' + port);
});
```

File: package.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "grad23-snakesandladders",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (Marco-Gomes-BBD) [inline package.json]: Add a start script so we can launch the program easily.
```suggestion
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "node index.js"
  },
```

File: index.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    res.sendFile(path.join(__dirname, 'html/pages/home_page.html'));
+});
+
+app.get('/game', function(req, res) {
+    res.sendFile(path.join(__dirname, 'html/pages/game_page.html'));
+});
+
+app.get('/player-select', function(req, res) {
+    res.sendFile(path.join(__dirname, "html/pages/player_select.html"))
+});
```
Comment (Marco-Gomes-BBD) [inline index.js]: `req` is unused, add an underscore, so everyone knows that's intentional.

**Review by Marco-Gomes-BBD — CHANGES_REQUESTED** [review]
Overall good.
Basically just formatting issues.

Just make the appropriate changes, and we can merge.

File: index.js (hunk 2 above)
Comment (hewitt-BBD) [reply index.js:1]: Added the lines and removed the ones at the top of the files

**Review by Marco-Gomes-BBD — APPROVED** [review]

**Pull request merged**